lessons/basics/functions.md

```diff
@@ -93,6 +93,29 @@ iex> Length.of [1, 2, 3]
 3
 ```
 
+### Function Naming and Arity
+
+We mentioned earlier that functions are named by the combination of given name and arity (number of arguments). This means you can do things like this:
+
+```elixir
+defmodule Greeter2 do
+  def hello(), do: "Hello, anonymous person!"   # hello/0
+  def hello(name), do: "Hello, " <> name        # hello/1
+  def hello(name1, name2), do: "Hello, #{name1} and #{name2}"
```

Comment:
@doomspork has the final say :)
I remember when I first came in, reading through the official docs they defined it quite early and explicitly and there was little ambiguity, so I just personally thought that style works well. But obviously thats just my opinion.